Misc/NEWS.d/next/Library/2020-09-20-15-14-05.bpo-41810.7l8lyV.rst

```diff
@@ -1,2 +1,3 @@
-:data:`types.EllipsisType`, :data:`types.EllipsisType` and
-:data:`types.NoneType` have been reintroduced.
+:data:`types.EllipsisType`, :data:`types.NotImplementedType` and
+:data:`types.NoneType` have been reintroduced, aforementioned types
+now being useable as annotations.
```

Comment:
I'm not trying to be overly picky here, but I really think you need to mention type checkers. If I was reading this comment 5 years from now, I'd think "but type(Ellipsis) was already usable as an annotation, so we should delete EllipsisType".

Reply:
Fair enough, fixed in https://github.com/python/cpython/pull/22336/commits/33caa6fb32ec98766a9402670176ddde16d47738.